packages/magic_ext/oauth/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: magic_ext_oauth
 description: A Magic oauth extension to support Magic social login authentication
-version: 0.3.6
+version: 1.0.0
```

Comment:
Let's keep `magic_ext_oauth` under 1.0 for now. Make it a minor bump (ie. `0.4.6`)